OWNERS_ALIASES

```diff
@@ -6,4 +6,45 @@ aliases:
   serving-api-reviewers:
   - mattmoor
   - tcnghia
-  - grantr
```

Comment:
:+1:

Reply:
I thought you might appreciate a spam reduction :D